### DISCUSSION TITLE
Possible to pass classNames to single elements?

### DISCUSSION THREAD

**Discussion by svenruegg** [JedWatson/react-select#5282] [Q&A]
I wondered if there is a way how i can pass classes to single elements of the select so that I can use a class based style approach like bootstrap or tailwind to style the select.

I tried to define a style but there I can not use my class based colors etc.

Thanks
Sven

**Comment by manjushsh** [comment]
@svenruegg 
this?
[Using classNames](https://react-select.com/styles#using-classnames)

**Reply by svenruegg** [reply]
not really. i mean that works for the top element but not for the childs.

something like this would be nice:
``
const customStyles = {
  option: (provided, state) => ({
    ...provided,
    className: 'bg-black text-left text-light pl-4',
  })
}``

but this leads to a css property classname instead of adding the classes
